nextcord/client.py

```diff
@@ -475,9 +497,31 @@ async def on_error(self, event_method: str, *args: Any, **kwargs: Any) -> None:
         overridden to have a different implementation.
         Check :func:`~nextcord.on_error` for more details.
         """
+
         print(f'Ignoring exception in {event_method}', file=sys.stderr)
         traceback.print_exc()
 
+    async def on_application_error(self, interaction: Interaction, exception: ApplicationError) -> None:
```

Comment:
`on_application_command_error` is a more suitable name for this, otherwise it's too generic and doesn't describe what it does